joomla/joomla-cms
# [6.0] Delete 5.2.2-2024-09-24.sql update SQL scripts

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) deleted the "5.2.2-2024-09-24.sql" update SQL scripts from the sources and adds them to the list of files to be deleted on update in the script.php file.

### Testing Instructions

Code review, or if you want to do a real test:

Update a recent 5.3-dev nightly build to the latest 6.0-dev nightly build to get the actual result, and update a recent 5.3-dev nightly build to the patched package created by Drone for this PR to get the expected result.

The latest 5.3-dev nightly build can be found here: https://developer.joomla.org/nightlies/Joomla_5.3.0-alpha1-dev-Development-Full_Package.zip

The latest 6.0-dev nightly build update package can be found here: https://developer.joomla.org/nightlies/Joomla_6.0.0-alpha1-dev-Development-Update_Package.zip

The patched package created by Drone can be found here: https://artifacts.joomla.org/drone/joomla/joomla-cms/6.0-dev/44512/downloads/80513/Joomla_6.0.0-alpha1-dev+pr.44512-Development-Update_Package.zip

### Actual result BEFORE applying this Pull Request

After an update from a recent 5.3-dev nightly build to the latest 6.0-dev nightly build, the files `administrator/components/com_admin/sql/updates/mysql/5.2.2-2024-09-24.sql` and `administrator/components/com_admin/sql/updates/postgresql/5.2.2-2024-09-24.sql` are present.

But that should not be the case, there should not be any 5.x update SQL scripts.

### Expected result AFTER applying this Pull Request

After an update from a recent 5.3-dev nightly build to the patched package created by Drone for this PR, there are no 5.x update SQL scripts in folders `administrator/components/com_admin/sql/updates/mysql` and `administrator/components/com_admin/sql/updates/postgresql`.

### Link to documentations
Please select:
